### PR TITLE
Electron - getting started copy

### PR DESCRIPTION
**Pull request by brianamarie**
**TL;DR**
This pull request adds the copy for the "Getting Started" with Electron course, and will be merged into `electron-course`. 

Originally, this had a goal ship date of Friday, July 28th. I'm not sure if that's still a realistic date, but any help is appreciated.  cc @github/services-training for direct commits, feedback on flow, and any general review or comments. 😊 

Context: 
- [Main tracking issue](https://github.com/github/services-training/issues/493) (<-In private repository, so all may not be able to see)
- [Project board](https://github.com/github/training-kit/projects/2)

We'll know this PR is done when there's words and base descriptions on each page. It doesn't have to be draft or final review ready, but enough for us to get through the course with directions.

### CODE REVIEW

**Comment by brianamarie** [comment]
Closing because this is against the wrong branch - apologies. :bow:

**Comment by muhaznazabdi10** [comment]
Hi buddy
